### ISSUE TITLE
DarnRead print statement

### ISSUE THREAD

**Issue by aburrell** [SuperDARN/pydarn#10]
the `pydarn.DarnRead` output doesn't have a user friendly object interface to let you know whether or not you successfully read in data.

```
import pydarn
fitacf_file = '20170120.0001.00.inv.fitacf'
sd_read = pydarn.DarnRead(fitacf_file)
print(sd_read)
<pydarn.io.superdarn.DarnRead object at 0x129385860>
```

**Comment by aburrell** [comment]
I don't mean a message that says something like 'completed plotting'.  I would like something that tells me what's in the object once it's read in.  If I do `print(sd_read)`, is there something sensible that will let me know what's inside (for example, the name of the file and if it is empty or not)?  If I'm doing some analysis, have to move to a different project, and then come back it can be helpful for me to pick up where I left off.

**Comment by kkotyk** [comment]
@aburrell I think you are asking for an implementation of __str__ or __repr__ correct? I think would be valuable.

**Comment by aburrell** [comment]
Yes, @kkotyk , exactly.

**Comment by mts299** [comment]
Sounds good, I will look into this once the PR #8 is merged into `develop` :)

**Comment by mts299** [comment]
Added this in, PR #8 is taking a bit longer than I would have hoped. 

```python
>>> import pydarn 
>>> darn_read = pydarn.DmapRead('20160331.2201.00.mcm.a.rawacf')
>>> darn_read 
DmapRead(20160331.2201.00.mcm.a.rawacf, 0, 0, 55576320)
>>> print(darn_read)
Reading from 20160331.2201.00.mcm.a.rawacf at cursor: 0 record number: 0 with a total number of bytes: 55576320
>>>
>>> darn_read = pydarn.DarnRead('20160331.2201.00.mcm.a.rawacf')
>>> darn_read 
DarnRead(20160331.2201.00.mcm.a.rawacf, 0, 0, 55576320)
>>> print(darn_read)
Reading from 20160331.2201.00.mcm.a.rawacf at cursor: 0 record number: 0 with a total number of bytes: 55576320
>>>
>>> rawacf_data = darn_read.read_rawacf()
>>>
>>> dmap_write = pydarn.DmapWrite(rawacf_data, 'test.rawacf')
>>> dmap_write
DmapWrite(test.rawacf, 0)
>>> print(dmap_write)
Writing to filename: test.rawacf
>>> print(dmap_write)
Writing to filename: test.rawacf at record number: 0
>>> 
>>> darn_write = pydarn.DarnWrite(rawacf_data, 'test.rawacf')
>>> darn_write
DarnWrite(test.rawacf, 0)
>>> print(darn_write)
Writing to filename: test.rawacf at record number: 0
```

I followed other styles of `__repr__` and `__str__` in python libraries.

**Comment by mts299** [comment]
#PR 8 was merged. Calling this issue closed and will add the `__str__` method a style requirement :+1:
